packages/functions/transform/src/config.ts

```diff
@@ -20,7 +20,6 @@ export type TransformEntities = {
   mergeRequests: typeof transform.mergeRequests;
   repositories: typeof transform.repositories;
   dates: typeof transform.dates;
-  mergeRequestFacts: typeof transform.mergeRequestFacts;
```

Comment:
This went from reports->facts->metrics ? I guess we want to export metrics table here